### PR TITLE
Add name of author to dump

### PR DESCRIPTION
**Pull request by 8sd**
Add name from `c.mail_from[0]` to the csv dump of the patches.

### CODE REVIEW

**Comment by rralf** [comment]
Could you please explain why you want to add this field? I can't see any users of that field.

**Comment by 8sd** [comment]
I need the name for the analyses if authors are discriminated.
The tool I use to guess sex/country is based on the name. 
Having this data would significantly simplify this analysis.

File: bin/pasta_evaluate_patches.py [hunk 1]
```diff
@@ -240,6 +240,7 @@ def dump_characteristics(repo, characteristics, ignored, relevant, filename):
 
                 row = {'id': message_id,
                        'from': mail_from,
+                       'from_name': c.mail_from[0],
```
Comment (rralf) [inline bin/pasta_evaluate_patches.py:243]: Guess that should be refactored to avoid confusion.
from -> from_email.

Comment (rralf) [reply bin/pasta_evaluate_patches.py:243]: And have a look at line 231. We can simplify it to sth. like:

from_email, from_name = c.mail_from.

**Comment by rralf** [comment]
I won't accept it unless we have upstream PaStA users of this value (e.g., in an R evaluation script). Adding superfluous values makes maintainence hard: At the moment, I can tell you with one look at the list which variables are used in any analysis, and which aren't. If we add optional values, this is not the case any longer. However, I'm willing to accept if we have users for the variable (with respect to the comments in the patch).

**Pull request closed without merging**